.github/workflows/zig-build.yml

```diff
@@ -39,9 +40,14 @@ jobs:
     # TODO: move from nightly to zig 0.12 final when released.
     steps:
     - uses: actions/checkout@v4
-    - name: "Install zig"
+    - name: "Install zig on Linux"
+      if: runner.os == 'Linux'
       run: |
         mkdir zig && curl https://ziglang.org/builds/zig-linux-x86_64-0.12.0-dev.2076+8fd15c6ca.tar.xz | tar Jx --directory=zig --strip-components=1
+    - name: "Install zig on MacOS"
+      if: runner.os == 'macOS'
+      run: |
+        mkdir zig && curl https://ziglang.org/builds/zig-macos-x86_64-0.12.0-dev.2076+8fd15c6ca.tar.xz | tar Jx --directory=zig --strip-components=1
```

Comment:
Can we define ZIG_XZ_URL (probably in lower case) above and use it in 2 install lines?

Reply:
Yes but I picked this style because I reckoned it would better fit on windows where I imagine the commands are entirely different (I haven't really checked, just assume it's different).

Reply:
Aha, maybe not URL but just zig_version (to have a single place where ther version is specified per file)

Reply:
> maybe not URL but just zig_version (to have a single place where ther version is specified per file)

I will change it myself